pkg/module/go_get_fetcher_test.go

```diff
@@ -32,8 +34,34 @@ func (s *ModuleSuite) TestGoGetFetcherFetch() {
 	r.NoError(err)
 	r.True(len(zipBytes) > 0)
 
-	r.NoError(ref.Clear())
+	// close the version's zip file (which also cleans up the underlying diskref's GOPATH) and expect it to fail again
+	r.NoError(ver.Zip.Close())
 	ver, err = ref.Read()
 	r.NotNil(err)
 	r.Nil(ver)
 }
+
+func ExampleFetch() {
+	repoURI := "github.com/arschles/assert"
+	version := "v1.0.0"
+	goBinaryName := os.Getenv("GO_BINARY_PATH")
```

Comment:
Here you can use (./pkg/config/environment).GoBinPath

Reply:
I did see that. Since this is meant to be an example in the godoc, I wanted to keep it really simple so the reader could understand how to use Fetch without looking anything else up.

Definitely open to switching to `GoBinPath()` if you believe that's better here.

Reply:
"GO_BINRAY_PATH" is an Athens convention anyway so it's probably ok to do `env.GoBinPath()`